components/home/resource-list.tsx

```
@@ -0,0 +1,101 @@
+import React from "react";
+import { RESOURCE_TYPES, ResourceTypeEnum } from "@/lib/constants";
+
+
+const resourceTypeLabels: Record<ResourceType, string> = {
```

ESLint

[error] 'resourceTypeLabels' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)